package.json

```diff
@@ -1,110 +1,120 @@
-      "git add"
-    ]
-  },
-  "name": "ng-chartist",
-  "peerDependencies": {
-    "@angular/common": ">=4.1.2",
-    "@angular/core": ">=4.1.2",
-    "@types/chartist": "^0.9.38",
-    "chartist": ">=0.9.8"
+  "homepage": "https://github.com/willsoto/ng-chartist",
+  "bugs": {
+    "url": "https://github.com/willsoto/ng-chartist/issues"
   },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/willsoto/ng-chartist.git"
   },
+  "license": "Apache-2.0",
+  "author": "Will Soto",
+  "files": [
+    "**/*.{js,ts,json,map}"
+  ],
   "scripts": {
-    "build": "yarn run build:lib && yarn run build:demo",
+    "build": "npm run build:lib && npm run build:demo",
     "build:demo": "ng build ng-chartist-demo --prod",
     "build:lib": "copyfiles package.json README.md projects/ng-chartist && ng build ng-chartist",
     "e2e": "ng e2e",
-    "gh-pages": "git checkout gh-pages && git merge master && yarn run build && yarn run doc && mv dist/ng-chartist-demo demo",
-    "lint": "yarn run lint:lib && yarn run lint:demo",
+    "gh-pages": "git checkout gh-pages && git merge master && npm run build && npm run doc && mv dist/ng-chartist-demo demo",
+    "postgh-pages": "git add . && git commit -m 'chore(*): build demo and docs' && git push && git checkout master",
+    "lint": "npm run lint:lib && npm run lint:demo",
     "lint:demo": "ng lint ng-chartist-demo",
     "lint:lib": "ng lint ng-chartist",
     "ng": "ng",
-    "pack": "cd dist/ng-chartist-lib && yarn pack",
-    "package": "yarn run build:lib && yarn run pack",
-    "postgh-pages": "git add . && git commit -m 'chore(*): build demo and docs' && git push && git checkout master",
+    "pack": "cd dist/ng-chartist-lib && npm run pack",
+    "package": "npm run build:lib && npm run pack",
     "start": "ng serve ng-chartist-demo",
     "test": "ng test ng-chartist --watch=false --browsers=ChromeHeadless",
     "doc": "compodoc projects/ng-chartist/src/lib -p projects/ng-chartist/tsconfig.lib.json -d docs",
     "release": "standard-version --sign --message 'chore(release): %s [skip ci]'",
-    "postrelease": "yarn build && npm publish dist/ng-chartist-lib && git push --follow-tags origin master"
+    "postrelease": "npm run build && npm publish dist/ng-chartist-lib && git push --follow-tags origin master",
+    "prepare": "husky install"
+  },
+  "commitlint": {
+    "extends": [
+      "@commitlint/config-conventional"
+    ]
+  },
+  "lint-staged": {
+    "*.{ts,js,md,json}": [
+      "prettier --write"
+    ]
+  },
+  "prettier": {
+    "arrowParens": "always",
+    "bracketSpacing": true,
+    "semi": true,
+    "singleQuote": true
+  },
+  "devDependencies": {
+    "@angular-devkit/build-angular": "~12.0.2",
+    "@angular-devkit/build-ng-packagr": "~0.1002.0",
+    "@angular/animations": "~12.0.2",
+    "@angular/cli": "~12.0.2",
+    "@angular/common": "~8.1.0",
+    "@angular/compiler": "~12.0.2",
+    "@angular/compiler-cli": "~12.0.2",
+    "@angular/core": "~8.1.0",
```

Comment:
And this? Did you go through all dependencies?

Reply:
I didn't do it manually, I used a [tool](https://github.com/raineorshine/npm-check-updates). I wonder why these weren't picked up